core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -327,7 +327,7 @@ public boolean isColocatedJoinsEnabled()
     }
 
     @Config("colocated-joins-enabled")
```

Comment:
Please add documentation for this config at `properties-optimizer.rst` just after `table-scan-node-partitioning-min-bucket-to-task-ratio`.